code/game/objects/items/tanks/tank_types.dm

```diff
@@ -160,15 +160,15 @@
 	icon_state = "emergency_engi"
 	worn_icon_state = "emergency_engi"
 	worn_icon = null
-	volume = 2 // should last a bit over 30 minutes if full
+	volume = 6 // should last a bit over 30 minutes if full
```

Comment:
Breath volume increased x4, tank capacity increased x3. Is this comment still correct?